templates/cassiopeia/scss/tools/variables/_variables.scss

```
@@ -95,6 +95,9 @@ $theme-colors: (
 $link-color:                          var(--cassiopeia-color-link);
 $link-hover-color:                    var(--cassiopeia-color-hover);
 
+// Breadcrumbs
+$breadcrumb-active-color:           $gray-700;
```

```suggestion
$breadcrumb-active-color:             $gray-700;
```
Would look nicer if aligned with the above assignments

---

I allow myself to directly commit this little code style change.